src/loaders/api/api.js

```diff
@@ -122,6 +122,22 @@ export function setAPI (agentIdentifier, forceDrain) {
     return appendJsAttribute('application.version', value, 'setApplicationVersion', false)
   }
 
+  apiInterface.start = (features, cb) => {
```

Comment:
Don't forget the angler update for the SM calling this API should generate.

Reply:
good call

Reply:
Does `agent-base` already need to include these fn for typing?

Reply:
yes, will need to update that as well

Reply:
updated typings, need to do angler PR still

Reply:
https://source.datanerd.us/agents/angler/pull/481